guides/common/modules/con_registering-hosts-and-setting-up-host-integration.adoc

```diff
@@ -1,4 +1,4 @@
-[id="registering_hosts_to_server_{context}"]
+[id="registering-hosts-and-setting-up-host-integration_{context}"]
```

Comment:
This broke UI links, see https://github.com/RedHatSatellite/foreman_theme_satellite/pull/90 / https://github.com/RedHatSatellite/foreman_theme_satellite/actions/runs/11230353017/job/31217571867?pr=90

Reply:
Adding a secondary ID might help. If not, I'll revert this change.

Reply:
I can also update the mapping in foreman_theme_satellite (but that'll require a new release etc).

Not sure what the preferred solution is. Keeping old ids (as secondary) seems like tech dept?

Reply:
I wouldn't mind keeping old IDs (although we might want to clean them up eventually in coordination with changes to the UI links). A bigger issue is that the secondary links seem to only work with RH d/s html-single builds, not the multi-page build linked from the UI.

I'm working on this in https://github.com/theforeman/foreman-documentation/pull/3358, which might indeed turn out to be a revert of the ID change (with a note as a warning for posterity because I'm pretty sure this would otherwise happen again).